style.css

```diff
@@ -99,3 +99,413 @@ header nav a {
 .socail-media-icon a {
   margin-right: 18px;
 }
+
+.wraper {
+  margin: 0 auto;
+}
+
+.works-container {
+  margin: 0 auto;
+  box-sizing: border-box;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: 1;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 100px;
+}
```

Comment:
- [OPTIONAL] I'd like to apologize as the previous reviewer forgot to mention this but as specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) you should use a `flexbox` to layout the items inside each project card. 🎯